docs/fsharp/tutorials/type-providers/creating-a-type-provider.md

```diff
@@ -1141,81 +1093,57 @@ type Service = ODataService<" https://services.odata.org/Northwind/Northwind.svc
-<br />  The `ProvidedTypes` support code has the following rules and limitations:
-<br />
-  1. Provided properties with indexed getters and setters aren't implemented.
-<br />
+### Limitations of the Type Provider Mechanism
 
-  2. Provided events aren't implemented.
-<br />
+The type provider mechanism in F# has the following limitations:
 
-  3. The provided types and information objects should be used only for the type provider mechanism in F#. They aren't more generally usable as System.Type objects.
-<br />
-
-  4. The constructs that you can use in quotations that define method implementations have several limitations. You can refer to the source code for ProvidedTypes-*Version* to see which constructs are supported in quotations.
-<br />
-
-- `Type providers must generate output assemblies that are .dll files, not .exe files.`
-<br />
+- The underlying infrastructure for type providers in F# doesn't support provided generic types or provided generic methods.
 
+- The mechanism doesn't support nested types with static parameters.
 
 ## Development Tips
 You might find the following tips helpful during the development process.
 
+### Run Two Instances of Visual Studio.
+
+You can develop the type provider in one instance and test the provider in the other because the test IDE will take a lock on the .dll file that prevents the type provider from being rebuilt. Thus, you must close the second instance of Visual Studio while the provider is built in the first instance, and then you must reopen the second instance after the provider is built.
+
+### Debug type providers by using invocations of fsc.exe.
 
-- `Run Two Instances of Visual Studio.` You can develop the type provider in one instance and test the provider in the other because the test IDE will take a lock on the .dll file that prevents the type provider from being rebuilt. Thus, you must close the second instance of Visual Studio while the provider is built in the first instance, and then you must reopen the second instance after the provider is built.
-<br />
+You can invoke type providers by using the following tools:
 
-- `Debug type providers by using invocations of fsc.exe.` You can invoke type providers by using the following tools:
-<br />
-  - fsc.exe (The F# command line compiler)
-<br />
+- fsc.exe (The F# command line compiler)
 
-  - fsi.exe (The F# Interactive compiler)
-<br />
+- fsi.exe (The F# Interactive compiler)
 
-  - devenv.exe (Visual Studio)
-<br />
+- devenv.exe (Visual Studio)
 
-  You can often debug type providers most easily by using fsc.exe on a test script file (for example, script.fsx). You can launch a debugger from a command prompt.
-<br />
+You can often debug type providers most easily by using fsc.exe on a test script file (for example, script.fsx). You can launch a debugger from a command prompt.
 
 ```
   devenv /debugexe fsc.exe script.fsx
 ```
 
   You can use print-to-stdout logging.
-<br />
 
 
 ## See Also
 [Type Providers](index.md)
```

Comment:
you'll need to add two spaces after this link so the new link shows up in a new line